index.js

```diff
@@ -10,35 +10,41 @@ import { normalMode, recreateMode, appendMode } from './modes';
       throw new Error(`Invalid repository: ${repository}`);
     }
 
+    const modes = Object.freeze({
+        NORMAL:     'normal',
+        APPEND:     'append',
+        RECREATE:   'recreate'
+    });
+
```

Comment:
could we move this to the `modes` module present in `modes.js` (probably at the top of the file) and import it from there?

(feels like a good fit to hold this enum)